### ISSUE TITLE
server crash when using tf2.cattr_starterpack.txt

### ISSUE THREAD

**Issue by Topyke** [nosoop/SM-TFCustomAttributeStarterPack#93]
When starting the server with tf2.cattr_starterpack.txt in the gamdata folder, my linux server crashes on startup
mmsource version: 2.0.0
sourcemod version: 1.12.0
The console says the following upon crashing 
cat: [hlds.1.pid](http://hlds.1.pid/): No such file or directory
email debug.log to [linux@valvesoftware.com](mailto:linux@valvesoftware.com)
when I remove the tf2.cattr_starterpack.txt from the gamedata folder, it stops crashing but most of the attributes stop working.
My debug.log:
```
----------------------------------------------
CRASH: Wed Sep 18 17:14:28 UTC 2024
Start Line: ./srcds_linux -nobreakpad -strictportbind -game tf +ip 209.192.176.150 -ip 209.192.176.150 -port 27015 +map pl_upward +maxplayers 24 -debug -verify_all
End of Source crash report
----------------------------------------------
```
I want to be able to use the full capabilities of the plugin but I have no idea how to fix this issue.

**Comment by nosoop** [comment]
Yeah; unfortunately as can be clearly seen, the plugin suite is struggling to stay current with TF2 releases now &mdash; the April 2024 game compiler toolchain update really took the wind out of the sails, and various issues with update tracking tooling made it a hassle to even start work.

11 months on I'm sure you've given up on trying to get this working (same here).  Though if not, the gamedata as of 87fdeb725dbcb95996d2607ff2b959a3d11f4366 should be current, though there may be issues with other plugins that remain since most of the plugins haven't been updated to account for the toolchain update.
